File: SortStringsByLength.py
# Sort Strings By Length
# - Given a list of strings `words`, sort them by their length in ascending order.
# - Constraints:
#   - 1 <= len(words) <= 10^4

from typing import List

class Solution:
    def sortByLength(self, words: List[str]) -> List[str]:
        return sorted(words, key=len)

File: SortStringsByLength.cs
// Sort Strings By Length
// - Given a list of strings, return the list sorted by their length in ascending order.
// - Time: O(n log n), Space: O(n)

using System;
using System.Linq;

public class Solution
{
    public string[] SortStringsByLength(string[] words)
    {
        return words.OrderBy(w => w.Length).ToArray();
    }
}
